scripts/socketserver/echo.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
```

Comment:
Interesting.... this adds another requirement of python to the dev setup....
I'd prefer to have this in node to reduce our requirements, but for a quick setup this is fine.

I don't know if all mac or windows installs have python3 by default (but Ubuntu does).

Maybe a PR in 0.12.0 to reduce this requirement (doesn't block 0.11 and not urgent til a dev complains, but a general feedback)